pygmy/model/link.py

```diff
@@ -284,6 +284,10 @@ def find(self, db, **kwargs):
         return self.link
 
     @dbconnection
-    def remove(self, db, long_url):
-        """But why?"""
-        pass
+    def remove(self, db, links):
+        short_codes = [link['short_code'] for link in links]
+        query = db.query(Link).filter(Link.short_code.in_(short_codes)).delete(synchronize_session=False)
```

Comment:
should we keep this filter query as well under the try block?

Reply:
Sure, makes sense